docs/partner_editable/faq_troubleshooting.adoc

```diff
@@ -18,4 +18,13 @@ For additional information, see https://docs.aws.amazon.com/AWSCloudFormation/la
 
 //== Troubleshooting
 
-//<Steps for troubleshooting the deployment go here.>
\ No newline at end of file
+//<Steps for troubleshooting the deployment go here.>
+
+
+//TODO Shivansh, Follow this FAQ, do you know why the guide includes only the parameters for launching into a new VPC?
+
+//TODO Shivansh, In Table 10, the five descriptions simply repeat the info in the labels. We'll want to define those five items instead, or say something helpful about each. (Please leave this comment in the doc until we've addressed this.)
+
+//TODO Shivansh, Table 11 mentions Availability Zones. Are they part of this Quick Start? If so, we need to review those descriptions. (Please leave this comment in the doc until we've addressed this.)
```

Comment:
I don't see availability zone in table 11 of the latest guide. Are you referring to any other table?

Reply:
This was previously Table 2. When I moved the parameter tables to the end as appendixes, this became Table 11.